cloud/cluster/fetch_cron.go

```diff
@@ -0,0 +1,38 @@
+package cluster
+
+import (
+	"fmt"
+	"time"
+)
+
+type FetchCron struct {
+	Ticker  *time.Ticker
+	f 	    Fetcher
+	Cl 		*Cluster
```

Comment:
We've configured pants to error on code that isn't gofmt'ed. (See also go vet, which complained about your first push)